app/controllers/accounts/contacts_controller.rb

```diff
@@ -69,33 +73,39 @@ def update
     if @contact.update(contact_params)
       flash[:notice] = 'Contato atualizado com sucesso!'
       if params[:contact][:deal_page_id]
-        redirect_to account_deal_path(current_user.account, params[:contact][:deal_page_id]) 
+        redirect_to account_deal_path(current_user.account, params[:contact][:deal_page_id])
       else
-      # redirect_to account_contacts_path(current_user) 
+        # redirect_to account_contacts_path(current_user)
         render :update, status: :ok
       end
     else
-      render :edit, status: :unprocessable_entity 
+      render :edit, status: :unprocessable_entity
     end
   end
 
   # DELETE /contacts/1 or /contacts/1.json
   def destroy
     @contact.destroy
     respond_to do |format|
-      format.html { redirect_to contacts_url, notice: "Contact was successfully destroyed." }
+      format.html { redirect_to contacts_url, notice: 'Contact was successfully destroyed.' }
       format.json { head :no_content }
     end
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_contact
-      @contact = Contact.find(params[:id])
-    end
 
-    # Only allow a list of trusted parameters through.
-    def contact_params
-      params.require(:contact).permit(:full_name, :phone, :email, custom_attributes: {})
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_contact
+    @contact = Contact.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def contact_params
+    params.require(:contact).permit(:full_name, :phone, :email,
+                                    custom_attributes: {}).merge(label_list: labels_list_formatted_params)
```

Comment:
Hmm será que algo nesse sentido não ficaria mais simples?

```suggestion
    params.require(:contact).permit(:full_name, :phone, :email,
                                    custom_attributes: {}, label_list:{})
```

```suggestion
    params.require(:contact).permit(:full_name, :phone, :email,
                                    custom_attributes: {}, label_list:[])
```